contracts/modules/TransferManager/GTM/GeneralTransferManager.sol

```diff
@@ -236,14 +247,14 @@ contract GeneralTransferManager is GeneralTransferManagerStorage, TransferManage
     }
 
     function _modifyTransferRequirements(
-        uint256 _transferType,
+        TransferType _transferType,
         bool _fromValidKYC,
         bool _toValidKYC,
         bool _fromRestricted,
         bool _toRestricted
     ) internal {
-        require(_transferType < 3, "Invalid TransferType");
-        transferRequirements[_transferType] =
+        require(uint8(_transferType) < 3, "Invalid TransferType");
```

Comment:
It can be removed

Reply:
why?

Reply:
It will be automatically checked as the `_transferType` value can't be 3 or more than 3 according to the enum declared